provider/testdata/src/paths.rs

```diff
@@ -19,6 +19,31 @@ pub fn uprops_toml_root() -> PathBuf {
     data_root().join("uprops")
 }
 
+/// Returns the absolute path to the collation tailoring TOML root directory.
+pub fn coll_toml_root() -> PathBuf {
+    data_root().join("coll")
+}
+
+/// Returns the absolute path to the collation diacritics TOML root directory.
+pub fn coll_dia_toml_root() -> PathBuf {
+    data_root().join("dia")
+}
+
+/// Returns the absolute path to the collation jamo TOML root directory.
+pub fn coll_jamo_toml_root() -> PathBuf {
+    data_root().join("jamo")
+}
+
+/// Returns the absolute path to the collation metadata TOML root directory.
+pub fn coll_meta_toml_root() -> PathBuf {
+    data_root().join("collmeta")
+}
+
+/// Returns the absolute path to the collation reordering TOML root directory.
+pub fn coll_reord_toml_root() -> PathBuf {
+    data_root().join("collreord")
+}
+
```

Comment:
Can test data use the same data structure as the uprops zip, i.e. having these directories inside `uprops` instead of as siblings?

Reply:
Used one sibling. It seems weird to put the `coll` directory inside `uprops` as they are logically two different things.

Reply:
Are these exported from ICU though? In that case we should restructure a bit, have a top-level icu_exportdata folder with properties and segmenter children. But we can do that in a followup, I'd suggest to just hardcode the test path for now